traffic_ops/app/db/migrations/2023012316280200_create_new_role.down.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+DELETE FROM public.role_capability rc WHERE rc.role_id='trouter';
```

Comment:
>> No need to delete from `role_capability` here, since `role_id` is a foreign key an `DELETE` cascasdes
>> 
>> https://github.com/apache/trafficcontrol/blob/396d474320f0628b72bb407efc795ac4cacde6cd/traffic_ops/app/db/create_tables.sql#L3772-L3773
>
> Removed line to delete on `role_capability`.  Assuming what you are say, that when it deletes `role` it will remove it from `role_capability` as well.

`DELETE FROM public.role_capability` came back and should be removed again 🙂

Reply:
Said I did it, but didn't.  D'oh